goth/runner/download/__init__.py

```diff
@@ -238,21 +239,20 @@ def download(
         :param output: directory to which the artifact should be extracted
         :param workflow_name: name of the workflow to select a run from
         """
-        logger.info(
-            "Downloading artifact. name=%s, branch=%s, commit=%s, workflow=%s",
+        logger.debug(
+            "ArtifactDownloader#download. name=%s, branch=%s, commit=%s, workflow=%s",
```

Comment:
It would be nice to have a convention for class/method names in log messages.

Reply:
Agreed, this one feels kind of "awkward". I try to stick with `{message}. var_name1=%s, ... var_nameN=%s` for `INFO` level logs, though I'm not sure about `DEBUG`. I can revert this back to `Downloading artifact.` if you think it's an outlier.